src/webrtc/call.ts

```diff
@@ -366,9 +370,15 @@ export class MatrixCall extends EventEmitter {
     public async placeVideoCall(): Promise<void> {
         logger.debug("placeVideoCall");
         this.checkForErrorListener();
-        const constraints = getUserMediaContraints(ConstraintsType.Video);
         this.type = CallType.Video;
-        await this.placeCallWithConstraints(constraints);
+        await this.placeCallWithConstraints(ConstraintsType.Video);
+    }
+
+    public createDataChannel(label: string, options: RTCDataChannelInit) {
```

Comment:
This method and corresponding event could be split into another PR if desired. I would like to reserve the last two parameters to `createGroupCall` for enabling datachannels for all calls and datachannel options though.

Reply:
I would argue for splitting this out as this PR is already large enough and the data channel stuff feels like a separate thing

Reply:
Started here https://github.com/matrix-org/matrix-js-sdk/pull/1929